SwiftGen.playground/Sources/SwiftGenStoryboardEnumBuilder.swift

```diff
@@ -14,30 +15,39 @@ public final class SwiftGenStoryboardEnumBuilder {
         
         class ParserDelegate : NSObject, NSXMLParserDelegate {
             var scenes = [Scene]()
+            var segues = [Scene]()
             var inScene = false
             var readyForFirstObject = false
+            var readyForConnections = false
             
             @objc func parser(parser: NSXMLParser, didStartElement elementName: String,
-                namespaceURI: String?, qualifiedName qName: String?,
-                attributes attributeDict: [String : String])
+              namespaceURI: String?, qualifiedName qName: String?,
+              attributes attributeDict: [String : String])
             {
-                
-                switch elementName {
-                case "scene":
-                    inScene = true
-                case "objects" where inScene:
-                    readyForFirstObject = true
-                case _ where readyForFirstObject:
-                    if let storyboardID = attributeDict["storyboardIdentifier"] {
-                        let customClass = attributeDict["customClass"]
-                        scenes.append(Scene(storyboardID, customClass))
-                    }
-                    readyForFirstObject = false
-                default:
-                    break
+              
+              switch elementName {
+              case "scene":
+                inScene = true
+              case "objects" where inScene:
+                readyForFirstObject = true
+              case _ where readyForFirstObject:
+                if let storyboardID = attributeDict["storyboardIdentifier"] {
+                  let customClass = attributeDict["customClass"]
+                  scenes.append(Scene(storyboardID, customClass))
                 }
+                readyForFirstObject = false
+              case "connections":
+                readyForConnections = true
+              case _ where readyForConnections:
```

Comment:
why not `case "segue" where readyForConnections`, just to be sure to only grab `segue` nodes here?

Note that I used `_` for the `objects` subsides because they can actually be of various type, like `viewController` or `tableViewController` etc, but I don't think that could actually be the case for `segue`… And I even wonder if that `<connection>` node doesn't also contain `IBOutlets` if any, may be worth adding some in the demo storyboard to check that out.